api/rhsm-subscriptions-api-spec.yaml

```diff
@@ -382,6 +504,73 @@ components:
               type: string
             gitHash:
               type: string
+    OptInConfig:
+      required:
+        - meta
+        - data
+      properties:
+        meta:
+          type: object
+          required:
+            - account_number
+            - org_id
+          properties:
+            account_number:
+              type: string
+            org_id:
+              type: string
+        data:
+          type: object
```

Comment:
Based on the example, I think you wanted an array here of OptInRecord (or some such)?